flexmeasures/api/dev/tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
+
+from flask_security import SQLAlchemySessionUserDatastore
+import pytest
+
+from flexmeasures.data.models.time_series import Sensor
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_api_test_data(db, setup_roles_users):
+    """
+    Set up data for API dev tests.
+    """
+    print("Setting up data for API v2.0 tests on %s" % db.engine)
+    add_gas_sensor(db, setup_roles_users["Test Supplier"])
+    give_prosumer_the_MDC_role(db)
```

Comment:
This line is currently obsolete (I checked) now that Prosumers can post sensor data, too (also line 28). But I think it shouldn't be obsolete, because we only want to let Prosumers post data to sensors they own themselves (we use the some rule for posting meter data).

Reply:
I implemented a placeholder for the ownership check. Currently sensors cannot have owners yet, because they are yet to be coupled to assets (which do have ownership).